pkg/cmd/component/cmd/ui/ui.go

```diff
@@ -403,3 +403,81 @@ func AddEnvironmentName() string {
 	handleError(err)
 	return strings.TrimSpace(environmentName)
 }
+func ApplicationOutputPath() string {
+	var path string
+	prompt := &survey.Input{
+		Message: "Provide a path to application folder",
+		Help:    "This is the path where application configuration is stored locally before you push it to the repository GitRepoURL",
```

Comment:
How about instead:

"This is the path where the GitOps configuration is stored locally before you push it to the repository GitRepoURL",